app/assets/stylesheets/application_static.scss

```diff
@@ -3,4 +3,4 @@
 @import 'app-bundle';
 
 // Non-webpack assets
-@import 'application_dev';
+@import 'application_hot';
```

Comment:
this name might be confusing -- it means 

`application_non_webpack_js_files`: only the JS files that are not provided by webpack.

compare to 

`application_static`: All JS files, including webpack and non-webpack. This is used for production, testing, and running the development rails server without the environment of `REACT_ON_RAILS_ENV=HOT`